code/modules/food&drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -162,6 +162,7 @@
 				icon_state = "tequilasunriseglass"
 				name = "tequila Sunrise"
 				desc = "Oh great, now you feel nostalgic about sunrises back on Terra..."
+			// Toxins Special is also in drinks.dm because shot glasses, mirror any changes there. //
```

Comment:
I believe changelogs are becoming more of a mandatory thing so you'll probably need one anyways.
And as with changes it'll probably take a little bit of time for people to learn, a changelog entry would help this.